functions/Install-DbaInstance.ps1

```diff
@@ -57,13 +62,15 @@ function Install-DbaInstance {
 
     .PARAMETER Authentication
         Chooses an authentication protocol for remote connections.
-        Allowed values: 'Default', 'Basic', 'Negotiate', 'NegotiateWithImplicitCredential', 'Credssp', 'Digest', 'Kerberos'
-        If the protocol fails to establish a connection
+        Allowed values: 'Default', 'Basic', 'Negotiate', 'NegotiateWithImplicitCredential', 'Credssp', 'Digest', 'Kerberos'.
+        If the protocol fails to establish a connection (HELP: What should be the end of this sentence?)
```

Comment:
So the idea is that it would request a confirmation for a failback authentication method, where it would create a PS Session Configuration on the remote server that would by default authenticate as specific credentials. Since this is considered insecure, there's an additional user configuration needed.